workspaces/rbac/plugins/rbac/src/components/ConditionalAccess/ConditionalAccessSidebar.tsx

```diff
@@ -76,27 +41,56 @@ export const ConditionalAccessSidebar = ({
   conditionRulesData,
   conditionsFormVal,
 }: ConditionalAccessSidebarProps) => {
-  const classes = useDrawerStyles();
-  const contentClasses = useDrawerContentStyles();
   return (
     <Drawer
       anchor="right"
       open={open}
       data-testid="rules-sidebar"
-      classes={{
-        paper: classes.paper,
+      sx={{
+        ['@media (max-width: 960px)']: {
+          width: '-webkit-fill-available',
+        },
+        width: '50vw',
+        height: '100vh',
+        gap: '3%',
+        display: '-webkit-inline-box',
```

Comment:
I see that you just moved the code. But anyway:

* Is this really necessary?
* Does this work well on Firefox?